### PR TITLE
Add Profile Type

### PR DESCRIPTION
**Pull request by povilasv**
REF https://github.com/conprof/conprof/issues/46

image

as  we are using special `__name__` label for profile types everything works, with current tsdb setup

Includes:
- Added UI rendering.
- Added Trace parsing, now Traces will be scrapped correctly instead of erroring out.
- Added Seconds argument to Trace/ Profile scrape jobs, which are set to their defaults in Go:

>Profile responds with the pprof-formatted cpu profile. Profiling lasts for duration specified in seconds GET parameter, or for 30 seconds if not specified. The package initialization registers it as /debug/pprof/profile. 
>Trace responds with the execution trace in binary form. Tracing lasts for duration specified in seconds GET parameter, or for 1 second if not specified. The package initialization registers it as /debug/pprof/trace.

### CODE REVIEW

File: scrape/scrape.go [hunk 1]
```diff
@@ -378,9 +378,24 @@ func (s *targetScraper) scrape(ctx context.Context, w io.Writer) error {
 		return fmt.Errorf("server returned HTTP status %s", resp.Status)
 	}
 
-	_, err = profile.Parse(io.TeeReader(resp.Body, w))
-	if err != nil {
-		return errors.Wrap(err, "failed to parse target's pprof profile")
+	switch profileType {
+	case ProfileTraceType:
+		// This is needed as trace.Parse fails with obscure error
+		// when you pass resp.Body
```
Comment (brancz) [inline scrape/scrape.go:384]: do we know why?

Comment (povilasv) [reply scrape/scrape.go:384]: I didn't dig deeply but the error is:

```
failed to parse target's trace profile: failed to read event 3 argument at offset 0 (failed to read trace at offset 13621: read 1, error EOF)
```

Comment (brancz) [reply scrape/scrape.go:384]: Odd, let’s try to come back to it. Could you open an issue for it?

Comment (povilasv) [reply scrape/scrape.go:384]: https://github.com/conprof/conprof/issues/60

**Pull request merged**